.gitpod.yml

```
@@ -6,7 +6,6 @@ tasks:
   - init: yarn && yarn build
     command: yarn dev --host 0.0.0.0
   - init: cd plugins/gatsby-remark-gitpod && npm install
```

Thanks! LGTM :+1: Maybe re-order this so that the terminal that runs `yarn dev` is not hidden?
```suggestion
  - init: cd plugins/gatsby-remark-gitpod && npm install
  - init: yarn && yarn build
    command: yarn dev --host 0.0.0.0
```